server/views/applications/pages/check-your-answers/review.njk

```diff
@@ -114,11 +114,11 @@
       {% for section in checkYourAnswersSections(page.application) %}
         <h2 class="govuk-heading-l">{{ section.title }}</h2>
         {% for task in section.tasks %}
-          <div class="main-box">
+          <div class="main-box" data-cy-check-your-answers-section="{{ task.id }}">
             <div class="box-title govuk-heading-m">
```

Comment:
Not sure but I wonder if a heading tag might be more relevant for the titles?

Reply:
That's probably true, I was going off the original designs

Reply:
Done this now!